NineChronicles.Headless/GraphTypes/ActionQueryFields/RapidCombination.cs

```diff
@@ -18,20 +19,20 @@ private void RegisterRapidCombination()
                     Name = "avatarAddress",
                     Description = "Avatar address to execute rapid combination"
                 },
-                new QueryArgument<NonNullGraphType<IntGraphType>>
+                new QueryArgument<NonNullGraphType<ListGraphType<NonNullGraphType<IntGraphType>>>>
```

Comment:
```suggestion
                new QueryArgument<NonNullGraphType<ListGraphType<IntGraphType>>>
```

ListGraphType을 쓰게 바뀌어야 할 것 같습니다.

Reply:
이미 해두셨네 이게 머여